src/hooks/useGetShoppingCart.ts

```
@@ -0,0 +1,33 @@
+import { useState, useCallback, useEffect } from 'react';
+import getShoppingCart from '../APIs/getShoppingCart';
+import { CartItem } from '../types/product.type';
+import { INITIAL_ERROR } from '../contexts/context.constant';
+import { Error } from '../types/product.type';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)